kahuna/public/js/components/gr-add-label/gr-add-label.html

```diff
@@ -1,9 +1,21 @@
 <button ng:class="{'small': ctrl.grSmall}"
-        ng:click="ctrl.addLabel()"
+        ng:click="ctrl.active = true"
         ng:disabled="ctrl.adding"
+        ng:if="!ctrl.active"
         title="Add label">
     <gr-icon ng:class="{'spin': ctrl.adding}">add_box</gr-icon>
-    <span ng:hide="ctrl.grSmall">
-        Add<span ng:show="ctrl.adding">ing...</span>
-    </span>
+        <span ng:hide="ctrl.grSmall">
+            Add<span ng:show="ctrl.adding">ing...</span>
+        </span>
 </button>
+<form class="gr-add-label__form" ng:if="ctrl.active" ng:submit="ctrl.addLabel()" >
+    <input type='text' ng:model="ctrl.newLabel" gr:auto-focus placeholder="label1, label2, label3…" ng:blur="ctrl.cancel()"></input>
```

Comment:
Can we wrap long lines with one attribute per line to make things more readable please?